mappings/oclc_bib.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime, timezone
+from uuid import uuid4
+
+from model import Record
+
+
+def map_oclc_bib_to_record(oclc_bib, related_oclc_numbers=[]) -> Record:
+    return Record(
+        uuid=uuid4(),
+        frbr_status='complete',
+        cluster_status=False,
+        source='oclcClassify',
+        # TODO: should this be an owi number? 
+        source_id=f"{oclc_bib['oclcNumber']}|oclc",
+        title=oclc_bib['title'],
+        # TODO: explicitly request subject search facet
+        subjects=[],
+        authors=[oclc_bib['creator']],
+        # TODO: split contributors from creator field
+        contributors=[],
+        languages=[oclc_bib['language']],
+        # TODO: previously we weren't pulling in dates but we could here
+        dates=[],
+        # TODO: should we include merged OCLC numbers? 
+        identifiers=[f"{oclc_bib['oclcNumber']}|oclc"] + \
+            map(lambda issn: f"{issn}|issn", oclc_bib['issns']) if oclc_bib['issns'] else [] + \
+            map(lambda isbn: f"{isbn}|isbn", oclc_bib['isbns']) if oclc_bib['isbns'] else [] + \
+            map(lambda oclc_number: f"{oclc_number}|oclc", related_oclc_numbers),
```

Comment:
What do you mean by 'merged' OCLC numbers exactly?

Reply:
Haha good question - theres a list of merged OCLC numbers that come back as part of the brief bib. 

Here's more info: https://help.oclc.org/Metadata_Services/WorldShare_Collection_Manager/Understand_record_processing/Criteria_for_new_update_delete_and_merged_records

Reply:
Oh ok I see now. I would say for right now we don't need to include the merged OCLC numbers since they are essentially two records put together and I'm not sure what the criteria for what makes two records merged according to OCLC.